test/Common/Spectra/gcm_standalone_visual_test.jl

```diff
@@ -91,8 +95,13 @@ contourf(
 )
 
 # Check magnitude
-println(sum(spectrum))
-println(sum(
-    0.5 .* var_grid[:, :, 1] .^ 2 *
-    reshape(sqrt.(1 .- sinθ .^ 2), (length(sinθ), 1)),
-))
+println(0.5 .* sum(spectrum))
+
+dθ = π .* wts ./ sum(wts)
+dθ = π / length(wts)
```

Comment:
this seems to be overriding the same variable value?

Reply:
An F for my review! :(

Reply:
Ahh well spotted, thanks! `wts` is 1 anyway, so either should work. All fixed! :)